Sanity/pkcs11/basic/runtest.sh

```diff
@@ -72,48 +72,52 @@ rlJournalStart
     rlPhaseStart FAIL "Simple text encryption and decryption (pin-source)"
         # TODO: Uncomment once implemention is finished:
         #   - THE PIN-SOURCE attribute not yet implemented
-
-        # rlRun "echo $PINVALUE > $PWD/pin"
-        # URI="{\"uri\": \"pkcs11:model=SoftHSM%20v2;manufacturer=SoftHSM%20project;serial=$TOKEN_SERIAL_NUM;token=$TOKEN_LABEL;id=$ID;module-path=$SOFTHSM_LIB?pin-source=$PWD/pin\", \"mechanism\": \"RSA-PKCS\"}"
-        # rlRun "echo 'this is a secret 2' > plain_text" 0 "Create a file to encrypt"
-        # rlRun "clevis encrypt pkcs11 '$URI' < plain_text > JWE" 0 "Encrypting the plain text"
-
-        # rlAssertDiffer JWE plain_text
-
-        # rlRun "clevis decrypt pkcs11 < JWE > decrypted_message" 0 "Decrypting the JWE"
-        # rlAssertNotDiffer decrypted_message plain_text
-        # rlRun "rm plain_text decrypted_message JWE $PWD/pin"
+        if [ "$PKCS11_IMPLEMENTED" -eq 1 ]; then
```

Comment:
Maybe you meant to have `-eq 0`  or "unset envvar". In the official released package this implementation is not there, yet I believe. I also think it is not present in the "developer" build either